src/index.html

```diff
@@ -46,7 +46,7 @@ <h2 class="subheading">Add your score</h2>
                     <input name="name" id="name" type="text" class="form_input" required maxlength="30"
                         placeholder="name" />
                     <input type="score" id="score" class="form_input" required placeholder="enter score here" />
```

Comment:
- [ ] Submitting big score broke the UI, you can add some kind of validation to prevent the user from submitting a long score
![Screenshot_20221015_171430](https://user-images.githubusercontent.com/25227022/195996903-4f6ac436-fb8c-47bc-b0a4-6036925601cf.png)